app/components/user-cards.js

```diff
@@ -59,23 +77,43 @@ export default Component.extend({
 
         this.set('sort_option','first_name')
         this.set('sort_order','asc')
+        this.set('sort_title','First Name')
 
     },
 
+    searchByEmployeeName(){
+        if (this.search !== '') {
+            this.set('displaysearch',this.search);
+            }
+        else {
+            this.set('displaysearch',null);
+        }
```

Comment:
The `displaysearch` servers the purpose with is done by  `search` itself. So, we don't need another variable.

Reply:
I missed the debounce part. Yes, we need to have a copy of search key.